### DISCUSSION TITLE
Bukkit Plugins in Arclight

### DISCUSSION THREAD

**Discussion by CondeCero** [IzzelAliz/Arclight#665] [Ideas]
How and Where  can i use bukkit plugins? (plugins like towny)

**Comment by heipiao233** [comment]
Did you see Arclight's logo at the top of the console log? If not, please check your run.bat because the Forge installer may modify it. Please write these commands in the file.
`@echo off`
`java -jar <your Arclight jar file>`
`pause`
Then use it like normal Spigot server. It may create a `plugins` folder.
# **_READ THE FAQ AT FIRST NEXT TIME!_**
